ep/build-velox/src/build_velox.sh

```diff
@@ -90,9 +91,12 @@ function compile {
     exit 1
   fi
 
-  COMPILE_OPTION="-DVELOX_ENABLE_PARQUET=ON"
-  if [ $ENABLE_BENCHMARK == "OFF" ]; then
-    COMPILE_OPTION="$COMPILE_OPTION -DVELOX_BUILD_TESTING=OFF -DVELOX_BUILD_TEST_UTILS=ON"
+  COMPILE_OPTION="-DVELOX_ENABLE_PARQUET=ON -DVELOX_BUILD_TESTING=OFF -DVELOX_BUILD_TEST_UTILS=OFF -DVELOX_ENABLE_DUCKDB=OFF"
```

Comment:
I suppose parquet compilation is not needed for ORC users. Better to keep it as optional.

Reply:
previous parquet option is enabled by default, I keep it as original.

> I suppose parquet compilation is not needed for ORC users

We should not separate users with file format, more is better.